src/services/emails/companyApplicationApproval.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable max-len */
+
```

Comment:
Yeah, the string interpolation in the html makes me kind of wary of template injection. We should definitely use some templating library that handles this before shipping this. Maybe this might be in a separate PR, but before having this go live with any sort of email-sending capabilities.

The templating would also be easier to grok than these strings, most likely.

Reply:
Someone in the board should also have a look at the emails, since they will be sent to companies. I think they are fine as-is, but they may benefit from some of their input or at least approval.

Reply:
Will add https://github.com/yads/nodemailer-express-handlebars.

@andrefmrocha if you could review the emails, that would be great. Otherwise just delegate someone instead :upside_down_face: